### ISSUE TITLE
image normalization with data augmentations

### ISSUE THREAD

**Issue by stephandooper** [stephandooper/ismi#12]
Data augmentation might influence the rescaling of the images somehow, check if this is true.
Furthermore, the images are not fully normalized (subtract training mean and standard deviation from the training, validation, and test set). This might again prove difficult if augmentations contaminate the result.

See 
[image normalization and rescaling](machinelearningmastery.com) with keras image data generators for guidance
